modules/snooker/src/main/ts/ephox/snooker/resize/BarManager.ts

```diff
@@ -132,8 +135,10 @@ export const BarManager = (wire: ResizeWire): BarManager => {
         * mouseout is not reliable within ContentEditable, so for all other mouseover events we clear bars.
         * This is fairly safe to do frequently; it's a single querySelectorAll() on the content and Arr.map on the result.
         * If we _really_ need to optimise it further, we can start caching the bar references in the wire somehow.
+        *
+        * Because the resizers were moved into the editor for inline mode, we need to check if the event target is not a resizer.
         */
-        if (SugarBody.inBody(event.target)) {
+        if (SugarBody.inBody(event.target) && !isResizer(event.target as SugarElement)) {
```

Comment:
You could avoid the magic cast here by instead doing `DomEvent.bind<MouseEvent, Element>` we know that it's a mouse event and we know that the nodes it could find could only be instances of Element. `event.target as SugarElement` makes it a `SugarElement<any>`.